### PR TITLE
Ac 128068239 add referral index filter presentation

### PR DESCRIPTION
**Pull request by NateOSI**

### CODE REVIEW

File: app/assets/javascripts/ReferralsIndexPage.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  constructor() {
+    super(...arguments)
+    this.state = {
+      referrals: [],
+    }
+  }
+
+  componentDidMount() {
+    const xhr = request('GET', '/referrals.json', null, null)
+    xhr.done((xhrResp) => {
```
Comment (pemaOSI) [inline app/assets/javascripts/ReferralsIndexPage.jsx]: Should we be making api call from React or have flux/redux architecture

File: app/views/layouts/application.html.haml [hunk 2]
```diff
@@ -1,8 +1,7 @@
 !!!
 %html
   %head
-    %meta{ :content => 'text/html; charset=UTF-8',
-      'http-equiv' => 'Content-Type' }/
+    %meta{charset: 'utf-8'}
     %title Casebook Intake Accelerator
     = stylesheet_link_tag gulp_asset_path('application.css'), defer: true
```
Comment (pemaOSI) [inline app/views/layouts/application.html.haml]: Can we change the title to Intake

**Comment by pemaOSI** [comment]
Looks good. left two comments

File: .pairs [hunk 3]
```diff
@@ -1,4 +1,6 @@
+<<<<<<< 8938d138d834704ec97d15838aacdfd260d4a3d1
 pairs:
```
Comment (NateOSI) [inline .pairs]: Good catch, must have happened in the rebase

**Pull request closed without merging**